quantization/nlp/bert/trt/e2e_tensorrt_bert_example.py

```diff
@@ -125,4 +243,32 @@ def get_next(self):
     quantizer.quantize_model()
     quantizer.model.save_model_to_file(qdq_model_path, False)
     print("QDQ model is saved to ", qdq_model_path)
-    
+
+    # QDQ model inference and get SQUAD prediction 
+    os.environ["ORT_TENSORRT_FP16_ENABLE"] = "1"  # Enable TRT FP16 precision
+    os.environ["ORT_TENSORRT_INT8_ENABLE"] = "1"  # Enable TRT INT8 precision
+    os.environ["ORT_TENSORRT_ENGINE_CACHE_ENABLE"] = "1"  # Enable TRT INT8 precision
+    batch_size = 1 
+    data_reader = BertDataReader(qdq_model_path, squad_json, vocab_file, batch_size, sequence_lengths[-1])
+    sess_options = onnxruntime.SessionOptions()
+    sess_options.graph_optimization_level = onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
+    session = onnxruntime.InferenceSession(qdq_model_path, sess_options=sess_options, providers=["TensorrtExecutionProvider", "CUDAExecutionProvider"])
+    all_predictions = inference(data_reader, session) 
+
+    prediction_file = "./prediction.json"
+    with open(prediction_file, "w") as f:
+        f.write(json.dumps(all_predictions, indent=4))
+        print("\nOutput dump to {}".format(prediction_file))
+
+    # Evaluate QDQ model for SQUAD v1.1
```

Comment:
why do we run evaluation in the separate script? Is it better to show evaluation result right after the inference is done since it's an e2e example?

Reply:
Yes, we can show result after the inference is done.
Will modify to execute these evalute-XXX.py directly inside our e2e example script.